moderation/Slowmode.go.tmpl

```diff
@@ -0,0 +1,60 @@
+{{/*
+	This command helps using a slowmode. it deletes a user's message if their slowmode isnt over people who you want can bypass slowmode as well as set it 
+ 
+	Usage: "-slowmode on (time in seconds) / -slowmode off "
+ 
+	Recommended trigger: ".*"
+	Trigger type: Regex
+ 
+	Credits:
+	Sponge :D
+*/}}
+ 
+{{/* CONFIGURATION VALUES START */}}
+{{$bypassperms := "ManageMessages"}}{{/* perms needed to bypass slowmode */}}
+{{$usageperms := "ManageMessages"}}{{/* perms needed for slowmode commands */}}
+{{/* put in the permissions you want the use to have in order to use or bypass the command */}}
+{{/* available perms: Administrator, ManageServer, ReadMessages, SendMessages, SendTTSMessages, ManageMessages, EmbedLinks, AttachFiles, ReadMessageHistory, MentionEveryone, VoiceConnect, VoiceSpeak, VoiceMuteMembers, VoiceDeafenMembers, VoiceMoveMembers, VoiceUseVAD, ManageNicknames, ManageRoles, ManageWebhooks, ManageEmojis, CreateInstantInvite, KickMembers, BanMembers, ManageChannels, AddReactions, ViewAuditLogs  */}}
+{{/* CONFIGURATION VALUES END */}}
+ 
+{{/* ACTUAL CODE */}}
+{{ $usageaccess := false }}
+{{ $bypassaccess := false }}
+{{ $viewperms := (split (index (split (exec "viewperms") "\n") 2) ", ") }}
+{{ if (in $viewperms $usageperms)}}{{ $usageaccess = true }}{{end}}
+{{ if (in $viewperms $bypassperms)}}{{ $bypassaccess = true }} {{ end }}
+{{if $usageaccess}}
+{{ if $matches := reFindAllSubmatches `\A-slowmode on (\d+)` .Message.Content }}
+{{$slowmodeduration := (index $matches 0 1)}}
+{{$db := (dbGet 660 "channels").Value}}
+{{with $db}}
+{{$value := sdict .}}
+{{$value.Set (str $.Channel.ID) (str $slowmodeduration)}}
+{{dbSet 660 "channels" $value}}
+{{else}}
+{{dbSet 660 "channels" (sdict (str $.Channel.ID) (str $slowmodeduration))}}
+{{end}}
+{{ sendMessage nil (print "done! slowmode has been set to `" $slowmodeduration "`") }}
+{{ else if reFind `\A-slowmode off` .Message.Content }}
+{{$db := (dbGet 660 "channels").Value}}
+{{if $db}}
+{{$value := sdict $db}}
+{{$value.Del (str $.Channel.ID)}}
+{{dbSet 660 "channels" $value}}
+{{end}}
+the slowmode has been removed from this channel
+{{ end }}
+{{ end }}
+{{ if not $bypassaccess}}
+{{$db := (dbGet 660 "channels").Value}}
+{{$value := sdict $db}}
+{{$get := $value.Get (str .Channel.ID)}}
+{{if $get}}
+{{ if $slowmode := dbGet .User.ID .Channel.ID }}
```

Comment:
Even though the key arg (`.Channel.ID`) is implicitly stringified here, better to be explicit.
```suggestion
{{ if $slowmode := dbGet .User.ID (str .Channel.ID) }}
```